services/data_plane_manager/src/main/resources/machine.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
What are those .jks files used for?

We could also remove machine.json. This file is not for host manager, not for DPM.

Reply:
would remove

Reply:
changed in 9a10de4